packages/amplify-e2e-tests/package.json

```diff
@@ -18,7 +18,10 @@
   "private": true,
   "scripts": {
     "e2e": "npm run setup-profile && jest --verbose",
-    "setup-profile": "ts-node ./src/configure_tests.ts"
+    "setup-profile": "ts-node ./src/configure_tests.ts",
+    "build": "tsc",
+    "clean": "rimraf lib tsconfig.tsbuildinfo",
+    "watch": "tsc --watch"
```

Comment:
We dont need this as the tests are run with ts-jest

Reply:
This I did to do a yarn build in amplify-e2e-tests, so that dont have to run yarn build on all the packages.

Reply:
We dont need to transpile typescript as jest will do it on the fly. If you have the transpiled version then jest will end up using it and you are forced to compile again

Reply:
Ahh I got it , thnaks Yathi